data/journeys/docsite-journeys.md

```diff
@@ -31,90 +31,113 @@ Milestone: Scale
 
 ## Novice
 
+Get started with Ansible box content.
+
 ```yaml
 Milestone: Need to learn the basics
   Understand the challenges of modern infrastructure management [content gap]
   Understand the fundamentals of Ansible automation
+
 Milestone: Set up an automation project
   Install the Ansible package
-  Create a basic inventory file
-  Run an ad-hoc command
-Milestone: Write first playbook
-  Create a hello world playbook
-  Run the playbook with the ansible-playbook command
+  Run your first ad hoc command in a few easy steps
```

Comment:
I think that we should keep the "expanded" journey details for clarity. I know on the docsite has "run your first ad-hoc command in a few easy steps" but that comprises lines that you removed.